recipes/notices/top_banner_info/top_banner_info.vue

```diff
@@ -58,11 +60,20 @@ export default {
 <style lang="less">
 .d-top-banner-info {
    &__left {
-     margin: var(--su4) 0 var(--su4) var(--su8);
+     margin: var(--space-300) 0 var(--space-300) var(--space-400);
      min-width: 20%;
   }
+  &__middle {
+     display: flex;
+     align-items: center;
+     gap: var(--size-300);
+  }
   &__right {
-     margin: var(--su4) var(--su12) var(--su4) 0;
+     display: flex;
+     align-items: baseline;
+     justify-content: flex-end;
+     gap: var(--size-300);
+     margin: var(--space-300) var(--su12) var(--space-300) 0;
```

Comment:
guessing there is no viable replacement for --su12?

Reply:
Actually good catch. I'm updating that to `16`. But to answer your question, it would've been `calc(var(--space-300) * 8)`